content/en/platform/corda/4.8/open-source/key-concepts-ecosystem.md

```diff
@@ -12,59 +12,40 @@ menu:
 tags:
 - concepts
 - ecosystem
-title: Network
+- network
+title: Networks
 ---
 
 
-# Network
+# Networks
 
 ## Summary
 
-* *A Corda network is made up of nodes, each of which runs an instance of Corda and one or more CorDapps*
-* *Communication between nodes is point-to-point, and does not rely on global broadcasts*
-* *Each node has a certificate that maps its network identity to a real-world legal identity*
-* *The network is permissioned, with access requiring a certificate from the network operator*
+* *People and businesses using Corda can communicate over a peer-to-peer network.*
+* *A Corda network is made up of nodes, which represent real-world legal identities. Each of which runs an instance of Corda and one or more CorDapps.*
+* *Nodes communicate point-to-point—not by global broadcast.*
+* *Each node has a certificate that maps its network identity to a real-world legal identity.*
+* *Corda networks are semi-private—you need a certificate from the network operator to join.*
```

Comment:
```suggestion
* *Corda networks are semi-private — you need a certificate from the network operator to join.*
```
I'd add spaces before and after an em dash. If our style guide says sth different, please ignore.

Reply:
American em dash styling doesn't use spaces. But appreciate the level of detail!